infrastructure/tari_script/src/script.rs

```diff
@@ -637,11 +631,11 @@ impl TariScript {
     fn handle_to_ristretto_point(&self, stack: &mut ExecutionStack) -> Result<(), ScriptError> {
         let item = stack.pop().ok_or(ScriptError::StackUnderflow)?;
         let scalar = match &item {
-            StackItem::Hash(hash) => hash.as_slice(),
+            StackItem::Hash(hash) => hash.as_slice(), // TODO: confirm this isn't problematic
```

Comment:
```suggestion
            StackItem::Hash(hash) => hash.as_slice(),
```
There is an issue for this, so I am deleting the todo as these just get left in code and forgotten about.

Reply:
Sounds good! For future reference, the issue is #5818.